test/gurka/test_gtfs.cc

```diff
@@ -494,7 +550,11 @@ TEST(GtfsExample, MakeTile) {
 // TODO: TEST THAT TRANSIT ROUTING IS FUNCTIONAL BY MULTIMOTDAL ROUTING THROUGH WAYPOINTS, CHECK THAT
 // THE TRIP TYPE IS A TRANSIT TYPE
 
-TEST(GtfsExample, DISABLED_testRouting) {
+TEST(GtfsExample, DISABLED_test_routing) {
+
+  // TODO: create a simpler map: currently somehow at edge label 8 or so, it connects back to edge
+  // label 3, which has the origin as predecessor.. smth is wrong with some directed edges.. possibly
+  // with kRail not having opposing edges?
```

Comment:
we're almost there. smth is messed up in the expansion, not sure yet what it is. I'll create a much simpler test map tmrw and iteratively make it more complex to see what's going on.